alexmenuet-alexmenuet-taulia/hw4/BookManager/app/models/book.rb

```diff
@@ -1,2 +1,13 @@
 class Book < ActiveRecord::Base
+    has_and_belongs_to_many :stores, :join_table => :joined_tables
+    
+    validates :release_date,
+        presence: { message: "New entry needs release date" }
+        
+    validates :title,
+        presence: { message: "New entry needs in Tilte" }
+        
+    validates :title,    
+        uniqueness: { message: "The Title already exists!" }
+    
```

Comment:
Looks like you found a DRY solution for notifying the user of individual validation issues. :+1: